tasks/object-detection-yolov8/main.py

```diff
@@ -138,9 +138,20 @@ def getPatience(taskRun: TaskRun) -> int:
     return 2 ** 64
 
 
+def verifySplitValid(validationSplit: float, datasetSize: int) -> None:
+    if not 0 <= validationSplit < 1:
+        raise ValueError(f">> [ObjectDetection] validationSplit parameter ({validationSplit}) must be between 0 and 1")
+
+    minSamplesForSplit = int(1 / min(validationSplit, 1 - validationSplit))
```

Comment:
Can you elaborate on how this works?

Reply:
It calculates the minimum number of samples to make the split work. So a valSplit of 0.2 would give a minSamplesForSpit = 5.
Do you think we should keep this check, or no?